### DISCUSSION TITLE
Saved to flash at FB, Count 9310, Bytes 409 after button pressed single

### DISCUSSION THREAD

**Discussion by smartboart** [arendst/Tasmota#19991] [Q&A]
### PROBLEM DESCRIPTION
Sometimes my Node MCU is not accessilble / working anymore till i make a hard reset. 
Thats why i implemented a sys log web server more debug to check whats going on.
Saw following entry when i use single press button.

NodeMCU-Heizungsraum ESP-CFG: Saved to flash at F7, Count 9306, Bytes 4096

see complete log below.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x ] Device used (e.g., Sonoff Basic): _____NodeMCU
- [ x] Tasmota binary firmware version number used: _____| 13.2.0(sensors)
-- | --

  - [ x] Pre-compiled
  - [ ] Self-compiled
- [ x] Flashing tools used: _____ota
- [ x] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:
{"NAME":"Generic","GPIO":[1,1,1,1,1,1,1,1,1,1,1,1,1,1],"FLAG":25965,"BASE":18}
{"Module":{"18":"Generic"}}
 {"GPIO0":{"257":"Relay_i2"},"GPIO1":{"261":"Relay_i6"},"GPIO2":{"258":"Relay_i3"},"GPIO3":{"256":"Relay_i1"},"GPIO4":{"608":"I2C SCL"},"GPIO5":{"640":"I2C SDA"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"259":"Relay_i4"},"GPIO13":{"260":"Relay_i5"},"GPIO14":{"160":"Switch1"},"GPIO15":{"162":"Switch3"},"GPIO16":{"161":"Switch2"},"GPIO17":{"4832":"ADC Button_i"}}
```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:
{"Rule1":{"State":"ON","Once":"OFF","StopOnError":"OFF","Length":144,"Free":367,"Rules":"on switch2#state=1 do publish stat/sonoff/RESULT {\"POWER10\":\"OFF\"} endon on switch2#state=0 do publish stat/sonoff/RESULT {\"POWER10\":\"ON\"} endon"}}

{"Rule2":{"State":"ON","Once":"OFF","StopOnError":"OFF","Length":37,"Free":474,"Rules":"ON button1#state=10 DO power1 2 ENDON"}}



```
- [x ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
 {"StatusSTS":{"Time":"2023-11-12T16:22:06","Uptime":"0T01:04:02","UptimeSec":3842,"Heap":19,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","POWER5":"OFF","POWER6":"OFF","Wifi":{"AP":1,"SSId":"michaswlan","BSSId":"DC:15:C8:2B:53:0F","Channel":1,"Mode":"11n","RSSI":64,"Signal":-68,"LinkCount":1,"Downtime":"0T00:00:03"}}}
```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:
Nov 12 15:52:48 NodeMCU-Heizungsraum ESP-WIF: Checking connection...
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-MQT: tele/sonoff/STATE = {"Time":"2023-11-12T15:53:04","Uptime":"0T00:35:00","UptimeSec":2100,"Heap":19,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER1":"ON","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","POWER5":"OFF","POWER6":"OFF","Wifi":{"AP":1,"SSId":"michaswlan","BSSId":"DC:15:C8:2B:53:0F","Channel":1,"Mode":"11n","RSSI":64,"Signal":-68,"LinkCount":1,"Downtime":"0T00:00:03"}}
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-MQT: tele/sonoff/SENSOR = {"Time":"2023-11-12T15:53:04","Switch1":"ON","Switch2":"ON","Switch3":"OFF","BME280":{"Temperature":19.5,"Humidity":59.7,"DewPoint":11.5,"Pressure":993.5},"PressureUnit":"hPa","TempUnit":"C"}
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-BTN: Button1 multi-press 1
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-RUL: BUTTON1#STATE=10 performs "power1 2"
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-SRC: Rule
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-CMD: Grp 0, Cmd 'POWER', Idx 1, Len 1, Pld 2, Data '2'
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-MQT: stat/sonoff/RESULT = {"POWER1":"OFF"}
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-MQT: stat/sonoff/POWER1 = OFF (retained)
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-MQT: stat/sonoff/RESULT = {"Button1":{"Action":"SINGLE"}}
Nov 12 15:53:06 NodeMCU-Heizungsraum ESP-CFG: Saved to flash at F7, Count 9306, Bytes 4096
Nov 12 15:53:07 NodeMCU-Heizungsraum ESP-WIF: Sending Gratuitous ARP
Nov 12 15:53:08 NodeMCU-Heizungsraum ESP-WIF: Checking connection...
Nov 12 15:53:28 NodeMCU-Heizungsraum ESP-WIF: Checking connection...
Nov 12 15:53:35 NodeMCU-Heizungsraum ESP-MQT: tele/sonoff/STATE = {"Time":"2023-11-12T15:53:34","Uptime":"0T00:35:30","UptimeSec":2130,"Heap":19,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","POWER5":"OFF","POWER6":"OFF","Wifi":{"AP":1,"SSId":"michaswlan","BSSId":"DC:15:C8:2B:53:0F","Channel":1,"Mode":"11n","RSSI":64,"Signal":-68,"LinkCount":1,"Downtime":"0T00:00:03"}}
Nov 12 15:53:35 NodeMCU-Heizungsraum ESP-MQT: tele/sonoff/SENSOR = {"Time":"2023-11-12T15:53:34","Switch1":"ON","Switch2":"ON","Switch3":"OFF","BME280":{"Temperature":19.6,"Humidity":59.8,"DewPoint":11.5,"Pressure":993.5},"PressureUnit":"hPa","TempUnit":"C"}
Nov 12 15:53:48 NodeMCU-Heizungsraum ESP-WIF: Checking connection...
Nov 12 15:54:05 NodeMCU-Heizungsraum ESP-MQT: tele/sonoff/STATE = {"Time":"2023-11-12T15:54:04","Uptime":"0T00:36:00","UptimeSec":2160,"Heap":19,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"POWER1":"OFF","POWER2":"OFF","POWER3":"OFF","POWER4":"OFF","POWER5":"OFF","POWER6":"OFF","Wifi":{"AP":1,"SSId":"michaswlan","BSSId":"DC:15:C8:2B:53:0F","Channel":1,"Mode":"11n","RSSI":64,"Signal":-68,"LinkCount":1,"Downtime":"0T00:00:03"}}
Nov 12 15:54:05 NodeMCU-Heizungsraum ESP-MQT: tele/sonoff/SENSOR = {"Time":"2023-11-12T15:54:04","Switch1":"ON","Switch2":"ON","Switch3":"OFF","BME280":{"Temperature":19.6,"Humidity":59.6,"DewPoint":11.5,"Pressure":993.5},"PressureUnit":"hPa","TempUnit":"C"}
Nov 12 15:54:07 NodeMCU-Heizungsraum ESP-WIF: Sending Gratuitous ARP
Nov 12 15:54:08 NodeMCU-Heizungsraum ESP-WIF: Checking connection...


```

### TO REPRODUCE
_Steps to reproduce the behavior:_
{"Button1":{"Action":"SINGLE"}}

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
Everything is working when {"Button1":{"Action":"SINGLE"}} will be carried out, but another thing happen too..
NodeMCU-Heizungsraum ESP-CFG: Saved to flash at F7, Count 9306, Bytes 4096

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

![grafik](https://github.com/arendst/Tasmota/assets/38385805/c1ae55dc-1872-446c-8d5c-4df6266ef63d)

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by sfromis** [comment]
The message you mention is in no way about a "problem"
``` 
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-BTN: Button1 multi-press 1
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-RUL: BUTTON1#STATE=10 performs "power1 2"
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-SRC: Rule
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-CMD: Grp 0, Cmd 'POWER', Idx 1, Len 1, Pld 2, Data '2'
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-MQT: stat/sonoff/RESULT = {"POWER1":"OFF"}
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-MQT: stat/sonoff/POWER1 = OFF (retained)
Nov 12 15:53:05 NodeMCU-Heizungsraum ESP-MQT: stat/sonoff/RESULT = {"Button1":{"Action":"SINGLE"}}
Nov 12 15:53:06 NodeMCU-Heizungsraum ESP-CFG: Saved to flash at F7, Count 9306, Bytes 4096
```
Your analog Button input is registering as "pushed", and therefore Tasmota triggers your rule to toggle Power2. And by default (unless you disable this), a power output (like a relay) status change is saved in flash, to enable it being available after restart.

This is unrelated to the device not being accessible, unless you have some borderline electrical situation starving the ESP8266 of power. And having relays are drawing a fair amount of power. Maybe more than is available without voltage drops.

**Comment by smartboart** [comment]
Ok, thanks for the quick reply... This was just a line in the logg which i didn´t understood. Thats why i pointed it to my unsolved problem regarding the device stops working.... So now i know it is not my problem and i have to wait until it happens again. maybe then i can see more in the debug log... thanks a lot.. i will close this discussion..
